internal/mock/storage.go

```diff
@@ -66,6 +66,19 @@ func (m *Storage) TxIterator(
 	panic("not implemented") // TODO: Implement
 }
 
+func (m *Storage) BlockReverseIterator(_, _ uint64) (storage.Iterator[*types.Block], error) {
+	panic("not implemented") // TODO: Implement
```

Comment:
Leftover?

Reply:
No, just implemented the mock as is implemented for the other Iterator.